packages/cashscript/test/e2e/P2PKH.test.ts

```diff
@@ -50,6 +54,48 @@ describe('P2PKH', () => {
       const txOutputs = getTxOutputs(tx);
       expect(txOutputs).toEqual(expect.arrayContaining([{ to, amount }]));
     });
+
+    it('should succeed when defining its own utxos', async () => {
```

Comment:
This test looks nice. But could you split it up into two separate 'should's? One for the failure (`tooHigh`), and one for the success?